script/constructron.lua

```diff
@@ -868,48 +827,50 @@ me.conditions = {
     ---@param job Job
     ---@return boolean | string
     deconstruction_done = function(job)

```

Comment:
the return value annotation seems to be incorrect now.
```suggestion
    ---@param job Job
    ---@return boolean?
    deconstruction_done = function(job)
```